### PR TITLE
Jb45810

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
The first commit belongs to other PR

**Comment by martyone** [comment]
Rebased, updated after VirtualMachine::Features addition + some improvements/fixes

File: src/libs/sfdk/virtualmachine.h [hunk 1]
```diff
@@ -84,6 +84,7 @@ class SFDK_EXPORT VirtualMachine : public QObject
         ShrinkStorageSize = 0x08,
         OptionalHeadless = 0x10,
         Snapshots = 0x20,
+        SwapMemory = 0x40,
```
Comment (vigejolla) [inline src/libs/sfdk/virtualmachine.h:87]: I don't quite follow here. We have VirtualMachine::Features::SwapMemory, and we have VirtualMachineInfo::swapSupported. They are almost the same but not quite - what's the difference?

Comment (martyone) [reply src/libs/sfdk/virtualmachine.h:87]: One is part of public API, the other is internal.

**Comment by vigejolla** [comment]
LGTM

**Pull request merged**